tasks/install_linux_repo.yml

```diff
@@ -94,7 +94,7 @@
 - name: Override systemd service params
   ansible.builtin.template:
     src: consul_systemd_service.override.j2
-    dest: /etc/systemd/system/consul.service.d/override.conf
+    dest: "/etc/systemd/system/{{ consul_systemd_service_name }}.service.d/override.conf"
```

Comment:
This causes an ansible-lint violation, since tempting Ansible task names only works if the template is at the very end of the string. You'll need to reword the task (and drop the quotes) to make this work.